server.js

```diff
@@ -0,0 +1,149 @@
+var requestProxy = require('express-request-proxy'),
+  express = require('express'),
+  pg = require('pg'),
+  pug = require('pug'),
+  port = process.env.PORT || 3000,
+  app = express();
+
+pg.defaults.ssl = true;
+
+///////////////////////////////////////////////////////////////////////
+
+var passport = require('passport');
+
+// This will configure Passport to use Auth0
+var strategy = require('./setup-passport');
+
+// Session and cookies middlewares to keep user logged in
+var cookieParser = require('cookie-parser');
+var session = require('express-session');
+
+app.use(express.static('./'));
+app.use(cookieParser());
+// See express session docs for information on the options: https://github.com/expressjs/session
+app.use(session({ secret: 'ab1FaBisHks9YbbHPhDV3iYfgMZ412Kw-87hQVsiYsqHIG_8gONEzNE4GYY-ZX6A', resave: false, saveUninitialized: false }));
+
+app.use(passport.initialize());
+app.use(passport.session());
+app.set('views', './view');
+app.set('view engine', 'pug');
+
+app.get('/callback',
+passport.authenticate('auth0', { failureRedirect: '/url-if-something-fails' }),
```

Comment:
`failureRedirect: '/url-if-something-fails'` 
This should probably redirect to an Auth Fail page, or at least a 404 (though that's not an appropriate error in this case.